modules/monitoring/application/forms/Command/Object/CheckNowCommandForm.php

```diff
@@ -21,6 +21,26 @@ public function init()
         $this->setAttrib('class', 'inline');
     }
 
+    /**
+     * SubmitButton Label
+     *
+     * @var SubmitButtonLabel
+     */
+    protected $submitButtonLabel = 'Check now';
```

Comment:
This must be somehow marked as translatable. Consider doing so in method `init()`.